react/forms/assets/page.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import {
+  Page,
+  PageSection,
+  PageSectionVariants
+} from "@patternfly/react-core";
+
+class AppPage extends React.Component {
+  render() {
+    const { children } = this.props;
+
+    const pageId = "main-content-page-layout-simple-nav";
```

Comment:
Do we need reference to simple-nav -- looks like it was copied from another example?